pkg/aws/snow.go

```diff
@@ -0,0 +1,55 @@
+package aws
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+)
+
+const (
+	snowEC2Port = 8243
+)
+
+func snowEndpoint(deviceIP string) *ServiceEndpoint {
+	return &ServiceEndpoint{
+		ServiceID:     "EC2",
+		SigningRegion: "snow",
+		URL:           fmt.Sprintf("https://%s:%d", deviceIP, snowEC2Port),
+	}
+}
+
+func withCustomCABundleFile(certsFile string) config.LoadOptionsFunc {
+	return func(o *config.LoadOptions) error {
+		caPEM, err := ioutil.ReadFile(certsFile)
```

Comment:
Can this be split into `WithCA` and `WithCAFromFile` where the latter calls the former?